### DISCUSSION TITLE
VS Code Extension to quickly generate Ray's syntax

### DISCUSSION THREAD

**Discussion by xitox97** [spatie/ray#715] [Show and tell]
Hi guys, I loved ray so much. I created VSCode extension to help quickly generate ray syntax. 

Just need to hightligh some variable or line, then choose any available command to generate the syntax 🔥 

Currently only support PHP & Laravel syntax.  More coming soon!

You guys can check it out here https://marketplace.visualstudio.com/items?itemName=FarhanHadi.quick-ray-dumper 🛸

**Comment by freekmurze** [comment]
That's very cool 👍

**Comment by miquelbrazil** [comment]
I definitely plan on checking this out when I get some time.
